include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -46,3 +46,8 @@ static const char* const kOrtSessionOptionsConfigSetDenormalAsZero = "session.se
 // "1": enable. ORT does fusion logic for QDQ format.
 // Its default value is "1"
 static const char* const kOrtSessionOptionsEnableQuantQDQ = "session.enable_quant_qdq";
+
+// It controls whether allow the thread to spin a few times before blocking
+// "0": thread will block if found no job to run
+// "1": default, thread will spin a number of times before blocking
+static const char* const kOrtSessionOptionsConfigAllowSpinning = "session.allow_spinning";
```

Comment:
This uses the same setting for both intra-op and inter-op threadpool. It should be easy to separate them into 2 configs.

Reply:
Done.